.github/workflows/build_bitacross.yml

```diff
@@ -0,0 +1,58 @@
+name: Build bitacross-worker
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: The commit SHA or tag for checking out code
+        default: ""
+        required: false
+
+env:
+  CARGO_TERM_COLOR: always
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  build-worker:
+    runs-on: tee-prod-builder
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.ref || github.ref }}
+          fetch-depth: 0
+
+      - name: Build local builder
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: bitacross-worker/build.Dockerfile
+          tags: local-builder:${{ github.event.inputs.ref || github.ref }}
+          target: builder
+          build-args: |
+            WORKER_MODE_ARG=offchain-worker
+            SGX_MODE=HW
```

Comment:
does GH actions run on machine with HW support? I thought with GHA we can only use software mode :thinking: